tox.ini

```diff
@@ -2,9 +2,9 @@
 min_version = 4.11
 
 env_list =
-    python{39,310,311}-django4.2-wagtail{5.2,6.1,6.2}
-    python{310,311,312}-django5.0-wagtail{5.2,6.1,6.2}
-    python312-django5.1-wagtail{6.2,main}
+    python{39,310,311}-django4.2-wagtail{5.2,6.2,6.3}
+    python{310,311,312}-django5.0-wagtail{5.2,6.2,6.3}
+    python312-django5.1-wagtail{6.3,main}
```

Comment:
let's add Python 3.13 in the mix too (and in CI, pyproject.toml PyPI classifiers).

other than that LGTM

Reply:
Thanks. The latest couple of commits have these changes.